src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/validators.py

```diff
@@ -0,0 +1,14 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+def get_expension_validator(expanded_arguments, model_type):
```

Comment:
Why are we doing the parameter patching and so forth in a validator? We don't do this anywhere else.